src/sentry/api/endpoints/codeowners/external_actor/user_details.py

```diff
@@ -27,7 +27,7 @@ class ExternalUserDetailsEndpoint(OrganizationEndpoint, ExternalActorEndpointMix
     }
     owner = ApiOwner.ENTERPRISE
 
-    def convert_args(  # type: ignore[override]
```

Comment:
I don't see any type fixes here :/ how come you could remove the comment? 😅

Reply:
The fix was upstream. Before, we use to two a mismatched function signature with the superclass